lib/node_modules/@stdlib/constants/float16/gamma-lanczos-g/package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "@stdlib/constants/float16/gamma-lanczos-g",
+  "version": "0.0.0",
+  "description": "Lanczos gamma parameter g.",
```

Let's match what we have in the `float32` and `float64` variants:

```suggestion
  "description": "Arbitrary constant `g` to be used in Lanczos approximation functions.",
```